src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -77,13 +77,16 @@ class GroupSimilarIssuesEmbeddingsEndpoint(GroupEndpoint):
     def get_formatted_results(
         self, responses: Sequence[SimilarIssuesEmbeddingsData | None], user: User | AnonymousUser
     ) -> Sequence[tuple[Mapping[str, Any], Mapping[str, Any]] | None]:
-        """Format the responses using to be used by the frontend."""
+        """
+        Format the responses using to be used by the frontend by changing the  field names and
+        changing the cosine distances into cosine similarities.
+        """
         group_data = {}
         for response in responses:
             if response:
                 formatted_response: FormattedSimilarIssuesEmbeddingsData = {
-                    "message": response["message_similarity"],
-                    "exception": response["stacktrace_similarity"],
+                    "message": 1 - response["message_distance"],
+                    "exception": 1 - response["stacktrace_distance"],
                     "shouldBeGrouped": "Yes" if response["should_group"] else "No",
```

Comment:
why don't we return a boolean instead of "yes" / "no" and have frontend handle display?

Reply:
I figured this function was formatting everything for the frontend anyways, might as well pass the frontend exactly what we want displayed

Reply:
i'd say it's generally good to let the frontend handle formatting in most cases